src/lib.rs

```diff
@@ -1370,6 +1370,11 @@ impl Build {
                     cmd.push_opt_unless_duplicate(format!("-O{}", opt_level).into());
                 }
 
+                if cmd.family == ToolFamily::Clang && target.contains("android") {
+                    // for compatibility with code that doesn't use predefined `__ANDROID__`
+                    cmd.push_opt_unless_duplicate("-DANDROID".into());
```

Comment:
Can you expand the comment here to point to official locations how the official tooling is passing a flag like this?

Reply:
I added links to last NDK release where `-DANDROID` is used.